FIXME

I think this should be reverted/deleted because
CS6716485 and patch_502L05_2019_0312_CS6716485.zip
should be fixing that. That's what I noticed on
vfe_master at least and the
patch_502L05_2019_0312_cs6646113 is already
including that fix supposedly (as per
patch.version).
--- a/kernel/linux-4.1/net/core/blog.c
+++ b/kernel/linux-4.1/net/core/blog.c
@@ -2395,6 +2395,22 @@ BlogAction_t blog_filter( Blog_t * blog_
     blog_assertr( (blog_p->rx.info.hdrs != 0), PKT_NORM );
 
     /*
+     * Skip DHCP acceleration -- FlowCache cannot properly accelerate DHCP flows, are
+     * there are no "flows" per se. The src/port is always the same, no matter the client
+     * The only difference is the MAC address, but that's L2
+     *
+     * On BCM systems this is typically done in eb/iptables by using the SKIPLOG action, but
+     * we don't have netfilter hooks in OVS
+     */
+     if (RX_IPV4(blog_p)
+         && (blog_p->key.protocol == IPPROTO_UDP)
+         && ((blog_p->rx.tuple.port.dest == htons(67))
+             || (blog_p->rx.tuple.port.source == htons(67))))
+     {
+         return PKT_NORM;
+     }
+
+    /*
      * E.g. IGRS/UPnP using Simple Service Discovery Protocol SSDP over HTTPMU
      *      HTTP Multicast over UDP 239.255.255.250:1900,
      *
